docs/en/operation/Metrics-System.md

```diff
@@ -120,9 +129,237 @@ Metrics in Alluxio have the following format for non-master node metrics:
+| Worker.<WORKER_HOSTNAME>.CapacityFree | Free capacity of this worker in bytes
+| Worker.<WORKER_HOSTNAME>.BlocksCached | Total number of blocks in Alluxio worker storages |
+| Worker.<WORKER_HOSTNAME>.BlocksAccessed | Total number of times blocks in this worker are accessed |
+| Worker.<WORKER_HOSTNAME>.BlocksCanceled | Total number of aborted temporary blocks |
+| Worker.<WORKER_HOSTNAME>.BlocksDeleted | Total number of deleted blocks |
+| Worker.<WORKER_HOSTNAME>.BlocksEvicted | Total number of blocks removed by this worker |
+| Worker.<WORKER_HOSTNAME>.BlocksLost | Total number of lost blocks |
+| Worker.<WORKER_HOSTNAME>.BlocksPromoted | Total number of blocks moved by clients from one location to another in this worker |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheRequests | Total number of async cache requests |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheDuplicateRequests | Total number of duplicate requests of caching the same block asynchronously |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheSucceededBlocks | Total number of blocks succeed in async cache |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheFailedBlocks | Total number of blocks failed to be cached asynchronously |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheUfsBlocks | Total number of async cache blocks which have local source |
+| Worker.<WORKER_HOSTNAME>.AsyncCacheRemoteBlocks | Total number of remote blocks to async cache locally |
+
+### Process Common Metrics
+
+The following metrics are collected on each instance (Master, Worker or Client).
+
+* JVM attributes
+
+| Metric Name | Description |
+|-------------------------|-----------------------------------------------------|
+| name | The name of the JVM |
+| uptime | The uptime of the JVM |
+| vendor | The current JVM vendor |
+
+* Garbage collector statistics
+
+| Metric Name | Description |
+|-------------------------|-----------------------------------------------------|
+| PS-MarkSweep.count | Total number of mark and sweep |
+| PS-MarkSweep.time | The time used to mark and sweep |
+| PS-Scavenge.count | Total number of scavenge |
+| PS-Scavenge.time | The time used to scavenge |
+
+* Memory usage
+
+| Metric Name | Description |
+|------------------------------|-----------------------------------------------------|
+| total.committed | The amount of memory in bytes that is guaranteed to be available for use by the JVM |
+| total.init | The amount of the memory in bytes that is available for use by the JVM |
+| total.max | The maximum amount of memory in bytes that is available for use by the JVM |
+| total.used | The amount of memory currently used in bytes |
+| heap.committed | The amount of memory guaranteed to be available |
+| heap.init | The amount of memory available at initialization |
+| heap.max | The maximum amount of memory that is available |
+| heap.usage | The amount of memory currently used in GB|
+| heap.used | The amount of memory that has been used |
+| non-heap.committed |  |
```

Comment:
are these purposefully left blank?

Reply:
Ideally, it should be filled, I will try to find the descriptions of those metrics, and if it cannot be found, I think it's fine to leave it blank. I saw some documents about metrics also left some meomory metrics blank

Reply:
sounds good. thanks!